src/components/AppServicesEmptyState/AppServicesEmptyState.stories.tsx

```diff
@@ -0,0 +1,152 @@
+import React from "react";
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import {
+  AppServicesEmptyState,
+  AppServicesEmptyStateVariant,
+} from "./AppServicesEmptyState";
+
+export default {
+  title: "Shared Components/EmptyState",
```

Comment:
There is already an "Empty states" section in Syorybook, no need to come up with a new one

Reply:
Agreed,  But I was thinking to keep in shared components category as it's shared component. 
Otherwise we can move. Let me know if need to move.

Reply:
You just need to change how stories are grouped. Something like `title: "Empty States/App Services/Description of the state"`